packages/client/TODO.md

```diff
@@ -0,0 +1,160 @@
+# Planned structure of low level SDK
+
+## Legend:
+
+- [A] authenticated
+- [P] paginated
+- [M] mutation
+- [Q] query
+- [H] helper (no API interaction)
+
+## Environments
+
+- polygon, mumbai, mumbaiSandbox
+
+## Client and Authentication
+
+- [x] LensClient.init(environment)
+- [x] [Q] client.generateChallenge(address)
```

Comment:
what's the reasoning for not "namespacing" these methods similar to the others,? 

for example:

`client.auth.generateChallenge()`

Reply:
we decided to merge LensClient and Auth responsibilities as auth is an essential first step before using more advanced features of the API, and its usage/lifecycle is different than other groups/endpoints (ie. profile or publication).